### ISSUE TITLE
Repo needs clarification

### ISSUE THREAD

**Issue by Woodyet** [papuSpartan/stable-diffusion-webui-distributed#32]
Am I right in saying: This repo does not allow for utilisation of multiple GPU's for creation of one image, it only helps with distributed compute over multiple batches of images.

**Comment by papuSpartan** [comment]
> it only helps with distributed compute over multiple batches of images.

Not exactly, each batch is distributed. So, if you set batch size to 4, that batch may be split in two to two separate instances of sdwui for example. So if you then set batch **count** to 2, that would then mean 4 images being sent to each instance over the course of 2 batches.

**Comment by papuSpartan** [comment]
> This repo does not allow for utilisation of multiple GPU's for creation of one image

As far as I know, this is a mostly unsolved problem unless you're willing to accept some tradeoffs. Stable diffusion is a very iterative process, so there have been special samplers and tools like tiled diffusion which parallelize single image generation. However, if you take tiled diffusion for example, it often results in the need for more cherry-picking due to the lack of perfect stitching between tiles.

Basically, if you want to create more high quality images more quickly, use this. If you want to make extremely large images which would normally cause a memory overflow, use a tiling solution.

(Btw, tiled diffusion **can** be centrally controlled using this extension, but you may run into memory ceilings if your cards have  differing memory sizes)

**Comment by Woodyet** [comment]
Okay thanks for the clarification, I appreciate what you say, and I understand the limitations. I wish to try and address the issue of multi-gpu on one image myself soon is all. I was poking around to see what exists, (I also wanted to make sure I wasn’t using the repo incorrectly).

Thanks for the pointers, and for your own work, whilst this is not what I was looking for in this specific moment, the master/slave node chaining is going to be super useful for me in the future… Keep up the good work! 😊
